src/NServiceBus.AcceptanceTesting/ContextAppender.cs

```diff
@@ -105,6 +105,12 @@ public void FatalFormat(string format, params object[] args)
         void Log(string message, LogLevel messageSeverity)
         {
             var context = ScenarioContext.Current;
+            if (context == null)
+            {
+                // avoid NRE in case something logs outside of a test scenario
+                Console.WriteLine(message);
```

Comment:
An alternative would be to write to TestContext given that we already reference NUnit (as an alternative)

Reply:
good idea 👍

Reply:
changed to using TestContext.